Data-Structures/src/test/java/Graph/GraphTest.java

```diff
@@ -94,4 +96,24 @@ public void bFSTraverse() {
         assertEquals("b", list.get(1).data);
         assertEquals("c", list.get(2).data);
     }
+
+
+    @Test
+    public void dfsTraverse()
+    {
+        Graph g = new Graph();
+        Vertex v1 = g.addNode("a");
+        Vertex v2 = g.addNode("b");
+        Vertex v3 = g.addNode("c");
+        g.addEdge(v1, v2, 3);
+        g.addEdge(v2, v3, 3);
+        g.addEdge(v1, v3, 3);
+        List<String> s = new ArrayList<>();
+        List<String> list = dFS(g.Vertices.get(0), s);
```

Comment:
I had to change this line to `g.dFS(g.Vertices.get(0), s)` in order to get it work